tests/logictest/suites/base/03_dml/03_0023_insert_into_array

```diff
@@ -389,11 +389,11 @@ ee ff
 statement ok
 CREATE TABLE IF NOT EXISTS t18(id Int, arr Array(Int64 Null)) Engine = Fuse;
 
-statement ok
+statement error 1010
```

Comment:
From `ok` to `1010` errors need comments here.

Reply:
ok,  I`ll do it latter:

``` ERROR 1105 (HY000): Code: 1010, displayText = Cast error happens in casting from Array(Nullable(UInt8)) to Array(Nullable(Int64)).``

Reply:
done